changelog/1195.trivial.rst

```diff
@@ -0,0 +1 @@
+Added tox as an extra requirement.
```

Comment:
```suggestion
Added `tox <https://tox.readthedocs.io/en/latest/>`_ as an extra requirement.
```

I really want to make you add `tox` to the `intersphinx_mappings` so this auto cross-references, but not necessary. 🤣

Reply:
Using intersphinx isn't a bad idea here since tox will come up in a bunch of places in the development guide.